carma_wm/src/SignalizedIntersectionManager.cpp

```diff
@@ -287,7 +298,6 @@ namespace carma_wm
     // check if it already exists
     for (auto sig_grp_pair : signal_group_to_exit_lanelet_ids_)
     {
-      ROS_DEBUG_STREAM("Creating signal for: " << (int)sig_grp_pair.first);
```

Comment:
Any particular reason this and the above debug log were removed?

Reply:
I deleted it by mistake. I added it back